raiden/ui/cli.py

```diff
@@ -275,12 +275,12 @@ def run(ctx, **kwargs):
 
         app_ = ctx.invoke(app, **kwargs)
 
-        app_.discovery.register(
+#         spawn address registration to avoid block while waiting for the next block
```

Comment:
My bad, I forgot to run linter before pushing the changes. I've fixed that, but then we've started discussing branches and other stuff with Lefteris. Should I push commit to my master this PR is based on, or create proper feature branch w/ new PR as it is done with the other PRs?